api/core/agent/cot_agent_runner.py

```diff
@@ -373,14 +374,21 @@ def _format_assistant_message(self, agent_scratchpad: list[AgentScratchpadUnit])
 
         return message
 
-    def _organize_historic_prompt_messages(self) -> list[PromptMessage]:
+    def _organize_historic_prompt_messages(self, prompt_messages: list[PromptMessage] = []) -> list[PromptMessage]:
```

Comment:
Hi, I found the name of `prompt_messages` is a little confusing, as it is only used to calculate the length of tokens in current session and I'm not sure about why this parameter is required at the beginning of review it, what about rename it to `current_session_messages` ?

Reply:
optimized